### DISCUSSION TITLE
flink-shaded-guava依赖冲突

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/flink-cdc#1551] [Q&A]
flinkcdc 2.2.1版本依赖了flink-shaded-guava的18.0-13.0版本，但是flink-1.14依赖了flink-shaded-guava的30.1.1-jre-14.0版本, 运行的时候用到了两个版本的特有的类，请问这个如何解决

**Comment by taylorThunderbolt** [comment]
使用胖包或者修改cdc依赖的guava版本然后手动编译
